metrics.md

```diff
@@ -0,0 +1,85 @@
+# Universal Search Metrics
+
+A summary of the metrics the Universal Search add-on and recommendation server will record.
+
+**Definitions:**
+
+- **URL bar** — the unified address + search bar at the top of the browser.
+- **Popup** — the popup that appears when you begin typing into the URL bar.
+- **Search** — the full lifecycle, beginning when the user types a key into the url bar, and ending when the user either chooses an item from or otherwise dismisses the popup.
+
+
+## Client metrics
+
+Metrics gathered from the Universal Search add-on will be reported to the Test Pilot add-on, which will augment that information and relay it to Firefox's telemetry system via `submitExternalPing`. This is not currently implemented on the Test Pilot side; those efforts are encompassed by [issue #234](https://github.com/mozilla/testpilot/issues/234).
+
+
+### Events
+
+Universal Search will record and report each of the following events:
+
+- User selects a recommendation.
+	- Time from search start to search completion.
+	- Length of query when selected.
+	- List of enhancers applied (e.g. `Object.keys(recommendation.enhancements)`.
+	- Value of the recommendation's `data-type` attribute. _(NB: this likely won't matter in early experiments. This is intended to allow explicit monitoring of qualitative types of results, e.g. "movie cards".)_
+- User enters a URL.
+	- Time from search start to search completion.
+- Use directly searches using their default search engine.
+	- Time from search start to search completion.
+	- Length of query.
+- User selects a suggested search term.
+	- Time from search start to search completion.
+	- Length of query.
+	- Levenstein distance between query and completed search term.
+- User selects a bookmark.
+	- Time from search start to search completion.
+	- Length of query.
+- Use selects an item from history.
+	- Time from search start to search completion.
+	- Length of query.
+
```

Comment:
Probably worth recording when the user chooses nothing. This will be especially important when we attempt to answer questions in our recommendations such as with a Weather card.